media/Config/index.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// before building DOM, this function guide which tool will be built
+// you can find makeSelectTagForImport and buildOptionDom in buildDom.js
```

Comment:
Has `buildDom.js` landed? If not, it would be better to land `makeSelectTagForImport` then proceed.

Reply:
ok, then i add `no merge` label. and after buildDom.js land, make request for review